Sources/Fakery/Generators/Address.swift

```diff
@@ -78,4 +83,30 @@ public final class Address: Generator {
   public func longitude() -> Double {
     return drand48() * 360.0 - 180.0
   }
+  
+  public func coordinate(latitude: Double, longitude: Double, inRadius radius: Double = 1000) -> FakeryCoordinates {
+    let y0              = latitude
```

Comment:
Could you align the code according to the project code style?

Reply:
Sure...